### DISCUSSION TITLE
win11中用外网域名可以正常挂载，但是用内网地址就无法挂载

### DISCUSSION THREAD

**Discussion by xmxu2022** [AlistGo/alist#2586] [Q&A]
内网地址，192.168.100.200:5244/dav，就无法挂载
但是用域名，baibai456.org:5244/dav，就可以正常挂载
请问大神是什么原因呢？

**Comment by ii1024** [comment]
1 我现在还没有备案域名 怎么用家里的外网地址访问呢？ IP：5244
2 是不是实名了 不备案 也是可以用域名解析到家里的外网地址呢？

**Reply by xmxu2022** [reply]
是的，你不开网站不用备案。用DDNS就可以用域名访问了，具体看你用啥设备，原理一样
